### PR TITLE
Running Behat tests on Sylius-Standard

### PR DESCRIPTION
**Pull request by pamil**
Related to Sylius/Sylius#5058.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
         },
         {
-            "name":     "Community contributions",
+            "name": "Community contributions",
             "homepage": "http://github.com/Sylius/Sylius/contributors"
         }
     ],
     "require": {
-        "php": ">=5.5.9",
+        "php": "^5.5.9|^7.0",
```
Comment (michalmarcinkowski) [inline composer.json]: `^5.6` :)

**Comment by pjedrzejewski** [comment]
Nice, thanks Kamil!

**Comment by Soullivaneuh** [comment]
@pamil What is the goal of `etc/build` folder? Do we need to keep it for our projects?

**Comment by pamil** [comment]
The Behat failure logs go to `etc/build` folder.

**Pull request merged**